app/views/layouts/application.html.erb

```diff
@@ -49,19 +49,13 @@
     <a href="#main-content" class="govuk-skip-link" data-module="govuk-skip-link">Skip to main content</a>
 
     <%= render partial: 'layouts/header' %>
+
     <div class="govuk-width-container">
-      <div class="govuk-phase-banner no-print">
-        <p class="govuk-phase-banner__content">
-          <strong class="govuk-tag govuk-phase-banner__content__tag ">
-            <%= t 'layouts.application.header.phase' %>
-          </strong>
-          <span class="govuk-phase-banner__text">
-            <%= t 'layouts.application.header.phase_banner_text_1' %>
-            <%= link_to_accessible t('layouts.application.header.feedback'), new_feedback_path, class: 'govuk-link' %>
-            <%= t 'layouts.application.header.phase_banner_text_2' %>
-          </span>
-        </p>
-      </div>
+      <%= govuk_phase_banner(
+        tag: {text: t(".header.phase")},
+        text: t(".header.phase_banner_html", link: govuk_link_to(t(".header.feedback"), new_feedback_path)),
```

Comment:
The only tangible change here is an omitted `aria-label="feedback"`, which seemed pretty redundant since the link text itself `feedback`.

Reply:
@RoseSAK should probably check this. Can the components not have arbitrary html attributes added to them in some way?

Reply:
Agree this can be removed 👍